sys/src/9/amd64/qmalloc.c

```diff
@@ -665,19 +655,22 @@ mallocinit(void)
 	print("base %#p ptr %#p nunits %u\n", tailbase, tailptr, tailnunits);
 }
 
-static int
-morecore(uint nunits)
-{
-	/*
-	 * First (simple) cut.
-	 * Pump it up when you don't really need it.
-	 * Pump it up until you can feel it.
-	 */
-	if(nunits < NUNITS(128*KiB))
-		nunits = NUNITS(128*KiB);
-	if(nunits > tailnunits)
-		nunits = tailnunits;
-	tailnunits -= nunits;
+static struct Allocator _qmallocAllocator = {
```

Comment:
All that being said, is it better to close this PR, since it clearly doesn't add any value yet, or to iterate on it?